books/projects/x86isa/machine/instructions/shifts-spec.lisp

```diff
@@ -149,14 +149,14 @@
                       (mbe
                        :logic (part-select
                                dst
-                               :low ,size-1 ;; (- ,size src)
+                               :low (- ,size src)
```

Comment:
Is this necessary? We are under the case `src = 1` (line 144), so `(- ,size src)` should be the same as `,size-1`. Perhaps the comment in the original line should be `;; = (- ,size src)`?

Reply:
Yeah looks like it doesn't do anything. I must have not realized that didn't change anything.